capstone/src/main/java/com/google/univiz/CollegeDataApi.java

```diff
@@ -0,0 +1,13 @@
+package com.google.univiz;
+
+import java.util.List;
+import java.util.Set;
+
+/**
+ * The CollegeDataApi interface is responsible for all REST requests and JSON parsing to the
+ * CollegeData class. CollegeDataApi is the only class that will communicate with an external API.
```

Comment:
```suggestion
 * CollegeData class.
```

I wouldn't make claims that might change in the future :)